fili-generic-example/README.md

```diff
@@ -0,0 +1,83 @@
+Fili Generic Loader Application
+==================================
+
+This is application is meant to be able to run and connect to any instance of Druid to show the basic metrics and dimensions.
```

Comment:
Is there a standard way fili accepts command line parameters other places in the code? Or some library you use for parsing them?
Just want to make sure before I write something specific for this